core/region.go

```diff
@@ -271,13 +271,13 @@ func (r *region) requestSpotPrices() error {
 		// spot market
 		price, err := strconv.ParseFloat(*priceInfo.SpotPrice, 64)
 		if err != nil {
-			logger.Println(r.name, "Instance type ", instType,
+			debug.Println(r.name, "Instance type ", instType,
```

Comment:
I think it's enough to log the resources that AutoSpotting *is* processing and omit logs for everything it's not.